docs/inst.rst

```diff
@@ -44,37 +44,43 @@ To enable the development and deployment of applications leveraging the NPU, you
    * - Dependencies
      - Version Requirement
    * - Visual Studio
-     - 2019
+     - 2022
    * - cmake
      - version >= 3.26
    * - python
-     - version >= 3.9 
+     - version >= 3.10 
+   * - opencv
+     - version == 4.6.0
    * - Anaconda or Miniconda
      - Latest version
-
-
 |
 |
+**Note:** Visual Studio 2022 Community edition, ensure "Desktop Development with C++" is installed
+
 
 .. _install-bundled:
 
 *****************************
 Install the Ryzen AI Software
 *****************************
 
-Before installing the Ryzen AI Software, ensure that all the prerequisites outlined previously have been met and that the Windows PATH variable is properly set for each component. For example, Anaconda requires following paths to be set in the PATH variable ``path\to\anaconda3\``, ``path\to\anaconda3\Scripts\``, ``path\to\anaconda3\Lib\bin\``. The PATH variable should be set through the *Environment Variables* window of the *System Properties*. 
-
-Download the :download:`ryzen-ai-sw-1.1.zip <https://account.amd.com/en/forms/downloads/ryzen-ai-software-platform-xef.html?filename=ryzen-ai-sw-1.1.zip>` Ryzen AI Software installation package and extract it. 
+Before installing the Ryzen AI Software, ensure that all the prerequisites outlined previously have been met and that the Windows PATH variable is properly set for each component. 
+For example, Anaconda requires following paths to be set in the PATH variable ``path\to\anaconda3\``, ``path\to\anaconda3\Scripts\``, ``path\to\anaconda3\Lib\bin\``. 
+The PATH variable should be set through the *Environment Variables* window of the *System Properties*. 
 
-Open an Anaconda or Windows command prompt in the extracted folder and run the installation script as shown below. Make sure to enter "Y" when prompted to accept the EULA. 
+Download the RyzenAI Software (MSI) installer :download:`https://xcoartifactory:443/artifactory/aie-ipu-prod-local/com/xilinx/ryzenai-installer/main/118/ryzenai-1.2.0.msi` 
```

Comment:
Private server URL?

Reply:
This is just a place holder, will be replaced with the public link for the update